01/app.js

```diff
@@ -2,20 +2,17 @@ document.addEventListener('DOMContentLoaded', init);
 
 function init() {
     const divList = document.querySelectorAll('div');
-    
-    setBorderColorAsync(divList[0], 'red', function() {
-        setBorderColorAsync(divList[1], 'blue', function() {
-            setBorderColorAsync(divList[2], 'green', function() {
-                console.log('finish');
-            });
-        });
-    });
+    setAllBorderColorsAsync(divList)
+}
 
+async function setAllBorderColorsAsync([box1, box2, box3]) {
+    await setBorderColorAsync(box1, 'red');
+    await setBorderColorAsync(box2, 'blue');
+    await setBorderColorAsync(box3, 'green');
 }
 
-function setBorderColorAsync(element, color, callback) {
+function setBorderColorAsync(element, color) {
     setTimeout(() => {
         element.style.border = `3px solid ${color}`;
-        callback();
     }, Math.random() * 3000);
-}
\ No newline at end of file
+}
```

Comment:
Bardziej chodziło mi o to, aby to zrobić przy pomocy funkcji bez użycia ES6+ :)

Reply:
Należałoby więc zawrzeć to w treści polecenia ;D

Reply:
Obawiam się, że było :p

Reply:
"Utwórz odpowiednie funkcje"
No to są - jedna asynchroniczna, a druga to ta, która już była, ale zmodyfikowana ;) Pozostaje dopisać, by nie korzystać z ES6+ :P